App/Whatsapp/whatsapp.js

```diff
@@ -5,34 +5,35 @@ const client = require('twilio')(accountSid, authToken);
 const whatsappNumbers = require('./numbers')
 
 // function that handles sending of the whatsapp messages
-function sendWhatsappMessage (whatsappNumbers) {
+function sendMessage(numbers) {
+    function sendWhatsappMessage(whatsappNumbers) {
 
-    // to handle multiple numbers we need to create this loop below
-    // sending a message to each number.
-    whatsappNumbers.forEach(num => {
-        // send message
-        client.messages
-            .create({
-                // body should be the message you are sending. you can insert anything you feel like
-                body: `Hello, this is an automated whatsapp message from Node webscrapper project. This message proves that the app is working fine`,
-                // for now we are using the twilio sandbox untill production then it can be changed
-                from: `whatsapp:${process.env.SANDBOX_WHATSAPP_NUMBER}`,
-                // THIS NUMBER IS KEPT AS AN ENVIRONMENTAL VARIABLE IN THE nodemon.json file, key is number
-                to: `whatsapp:${num}`
-            })
-            // resolve
-            .then(message => {
-                // message was sent successsfully
-                console.log(message._solution)
-            })
-            .catch(error => {
-                // an error occured and message was not sent
-                console.log(`An error occured ${error}`)
-            })
-            .done();
-    })
-}
+        // to handle multiple numbers we need to create this loop below
```

Comment:
The indention of line `11,12,13` Should be left as before. This is beacuse lines with comments are not indented that goes for functions as well